config/metric_config_sample.yaml

```diff
@@ -36,6 +36,8 @@ output:
       saveMode: Overwrite
       # path of the file (can be omitted if using tableName and writing to warehouse)
       path: file.parquet
+      # Enable adding current timestamp suffix to path of the file
+      addTimestampToPath: True
```

Comment:
shouldn't it be true and not True?
maybe it's better to flag it as "createUniquePath" ?

Reply:
OK, no problem